zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/InterpreterFactoryTest.java

```diff
@@ -109,6 +109,10 @@ public void testFactoryDefaultList() throws InterpreterException, IOException {
     assertEquals(2, all.size());
     assertEquals("mock1", factory.get(all.get(0)).getName());
     assertEquals("a mock", factory.get(all.get(1)).getName());
+
+    // add setting with null option & properties expected InterpreterException.class
+    factory.add("a mock", "mock2", null, new Properties());
+    factory.add("a mock", "mock2", new InterpreterOption(false), null);
```

Comment:
these should be separate tests or have two try/catches, since the test where the properties argument is null will never run.